cms/static/cms/sass/settings/_cms.scss

```diff
@@ -342,7 +342,7 @@ $dialog-shadow: $modal-shadow;
 // Toolbar messages + screenblock + login form
 $messages-width: 300px;
 $messages-padding: 6px 10px 8px;
-$messages-color: $white;
+$messages-color: white;
```

Comment:
Is this because the messages bar always has the same background colour?

Reply:
Exactly: It is always shadow color and this is also dark in dark mode.